lib/network_linyar/newyork_state.dart

```diff
@@ -0,0 +1,30 @@
+part of 'newyork_bloc.dart';
+
```

Comment:
change the name of all states ex.
NewyorkSuccess ==> FetchedArticleSuccessState